client/src/app/components/stardust/TransactionPayload.scss

```diff
@@ -83,7 +83,7 @@
     }
   }
 
-  .transaction-from.card--content .card--content__output {
+  .transaction.card--content .card--content__output {
```

Comment:
Maybe we could change the `transaction` class name to  `transaction-payload_outputs` so its in line with BEM naming convention.